libquickevent/libquickeventcore/src/si/punchrecord.cpp

```diff
@@ -10,8 +10,8 @@ namespace quickevent {
 namespace core {
 namespace si {
 
-const QString PunchRecord::MARKING_RACE = QStringLiteral("race");
-const QString PunchRecord::MARKING_ENTRIES = QStringLiteral("entries");
+const QString PunchRecord::MARKING_READOUT = QStringLiteral("race");
```

Comment:
readout je to vzdycky, `RACE` rika, ze se ma tem punch pouzit ve fazi zavod, analogicky 'ENTRIES' znamena, ze ve fazi prihlasky

Reply:
MR řeší #492, pokud je přepínač v módu `Entries`, je automaticky zapnuto edit on punch (pokud se nepletu, nemá tento mód více využití), pokud je přepínač v módu `Race`, pak probíhá normální vyčítání.

Protože se lehce mění význam přepínače, přejmenoval jsem více významově i příslušné konstanty.

---
RACE je celý závod, neříká žádný časový údaj ani neurčuje nějakou podmnožinu, READOUT mělo být vyčtení doběhnuvšího závodníka, je možno najít lepší označení

Pokud přepínač nyní bude způsobovat editaci zavodníka při oražení krabičky, nedělal bych popis zbytečně více obecný, proto EDIT_COMPETITOR

Reply:
Jak jsem jiz psal, poradani zavodu ma nekolik fazi, prvni je prihlasovani a potom prichazi zavod, oddeluje se to ukoncenim prezentace, tohle nebudu menit. Marking `ENTRIES` rika, ze se ma punch pouzit ve fazi prezentace, prijde mi to logicke. Nerikam, ze to co navrhujes je spatne, ale neni to ani lepsi a nebudu prijimat zmeny, ktere delaji neco, co uz mame jinak, to bychom se mohli zacyklit.

Jinak IMO navrhovana zmena zpusobi, ze dialog stejne vyskoci na vsech pocitacich, jenom uz nepujde vypnou checkboxem `edit on punch`, takze je to podle me zmena k horsimu.

Reply:
Ano, neuvědomil jsem si že by případně bylo potřeba nejdříve vyřešit vyskakování edit okna jen na pripojenem pocitaci.

---
Chápu vnitřní rozdělení na entries a race. Moje iniciativa vyšla čistě z toho že mi přijde hloupe|zbytecne|neintuitivní mít v GUI přepínač Race <-> Entries, když jeho jediná funkce je, že po přepnutí můžu někde jinde, v jiném okně, jiným přepínačem zapnout edit on punch.

Reply:
je mi to jasny, tady jsem myslel trochu dopredu a to se nema delat. Kdyz jsem to vymyslel, tak byly jeste izolovane pluginy a nevedel jsem, co vsechno se s tim bude podnikat. Ted to dopadlo takhle a v pripade, ze smazeme checkbox `edit on punch`, nikdo uz nepozna, co se prepnutim na marking `ENTRIES` zapina za funkci. QE v tomhle pripade dosel jinam, nez jsem odhadoval. Nabizi se pojmenovat `marking race` na `edit on punch` a pak ma smysl prejmenovat i tu globalni promennou, jak je v tvem MR. Jenom mi to porad nevoni. Navrhuji vrati se k puvodni issue, vymyslet, co s tim a pak to naimplementovat.

Reply:
Přiznávám, že takto se případně zcela mění původní význam přepínače. Na druhou stranu jsem byl nadšený, když jsem zjistil, že do databáze se uloží i to v jakém módu byl čip načten. 

Ještě než to zavřu, abych měl celkovou představu, co se prosím nyní přesně mění (za proměné/stavy) uvnitř QE, při přepnutí mezi `Race <-> Entries`?

Reply:
uz jsem neco napsal do #502